server/src/main/scala/actors/PortStateActor.scala

```diff
@@ -46,15 +46,49 @@ class PortStateActor(liveStateActor: ActorRef,
 
   implicit val timeout: Timeout = new Timeout(30 seconds)
 
+  val liveCrunchStateActor: ActorRef = context.actorOf(liveCrunchStateProps, name = "crunch-live-state-actor")
+  val forecastCrunchStateActor: ActorRef = context.actorOf(forecastCrunchStateProps, name = "crunch-forecast-state-actor")
+
   val state: PortStateMutable = PortStateMutable.empty
 
-  val killActor: ActorRef = context.system.actorOf(Props(new RequestAndTerminateActor()))
+  val killActor: ActorRef = context.system.actorOf(Props(new RequestAndTerminateActor()))//, "port-state-kill-actor")
```

Comment:
we should look at making these names work.

Reply:
noted